src/Summoner/Template/Doc.hs

```diff
@@ -17,9 +17,12 @@ docFiles :: Settings -> [TreeFs]
 docFiles Settings{..} =
     [ File "README.md" readme
     , File "CHANGELOG.md" changelog
-    , File "LICENSE" $ unLicense settingsLicenseText
-    ] ++ maybeToList (File "CONTRIBUTING.md" <$> settingsContributing)
+    ] ++ [File "LICENSE" (unLicense settingsLicenseText) | isNotNoneLicense]
```

Comment:
Maybe we can use `++` at the end of the line here as well? With this way of specifying lists I kinda like:

```haskell
[ Foo 
, Bar ] ++
[ Baz ] ++
```
more than
```haskell
[ Foo 
, Bar 
] ++ [ Baz ] ++
```